src/base/static/components/molecules/map-legend-item.js

```diff
@@ -33,17 +25,12 @@ const MapLegendItemContainer = styled("div")({
 const MapLegendItem = props => {
   return (
     <MapLegendItemContainer>
-      {props.icon && <LegendIcon icon={props.icon} />}
-      {props.swatch && <ColorSwatch color={props.swatch} />}
+      <LegendIcon icon={props.icon} />
```

Comment:
For our new SurveyForm models, I dropped support for `swatch` and made the `icon` attribute mandatory.